apps/examples/board_specific_demo/ameba_mipi_demo/example_mipi_show.c

```diff
@@ -0,0 +1,11 @@
+extern MIPIDemoShow(void);
```

Comment:
Last comment.
Could you add including necessary headers?
```
#include <tinyara/config.h>
#include <stdio.h>
```

Reply:
Dear @sunghan-chang 
Included the necessary headers as requested.